lib/experian/precise_id/client.rb

```
@@ -18,9 +18,23 @@ def send_answers(options = {})
 
       def submit_request(request)
         raw_response = super
-        log_raw_response(raw_response)
         response = Response.new(raw_response.body)
+        check_response_for_errors(response, raw_response)
         [request,response]
+      rescue => e
```

Is swallowing all errors the intended behavior, or do they need to be reraised?